systemvm/patches/debian/config/opt/cloud/bin/cs/CsAddress.py

```diff
@@ -407,10 +410,10 @@ def fw_vpcrouter(self):
                             ])
 
         if self.get_type() in ["public"]:
-            self.fw.append(["nat", "front",
-                            "-A POSTROUTING -o %s -j SNAT --to-source %s" %
-                           (self.dev, self.address['public_ip'])
-                            ])
+            # self.fw.append(["nat", "front",
```

Comment:
You can but you can also make a new pr inlcuding @isoutham code (with attribution intact) and add ammending commits
My worry is: will this break any existing tests. prod code is safe so I won't -1, just looking for loopholes